packages/server/__tests__/autopauseUsers.test.ts

```diff
@@ -37,7 +37,7 @@ test('Autopause users who where never active', async () => {
     data: {
       user: {
         id: userId,
-        inactive: true
+        inactive: false // user will have lastSeenAt populated upon sign up so won't be inactive
```

Comment:
+1 This invalidates the test since it's not verifying that autopauseUsers actually pauses some users. That's ok for now, but could you add a comment to the test?